Tasks/Xcode/xcode.ts

```diff
@@ -439,7 +442,7 @@ async function run() {
         tl.setResult(tl.TaskResult.Succeeded, tl.loc('XcodeSuccess'));
     }
     catch (err) {
-        tl.setResult(tl.TaskResult.Failed, err);
+        tl.setResult(tl.TaskResult.Failed, err.message);
```

Comment:
You just log want to log `err` here.  Existing code that is not throwing `Error` will still work, and `Error` will get converted to string.

Reply:
It changed the error message (added Error: )but looks good otherwise. I updated the impacted test as well.